Updating sites module (#1258)
Azure/ResourceModules: docs/wiki/Getting started - Scenario 1 Onboard module library and CI environment.md

> **Note**: For details about the dependencies pipeline design please refer to the dedicated [Dependencies pipeline design](./The%20CI%20environment%20-%20Pipeline%20design.md#dependencies-pipeline) section.

> ## 4.1 Manual Dependencies

Reviewer: ```suggestion
## 4.1 Manual Dependencies

In special cases, manual actions may be required to provision certain resources that are not covered by our dependency pipeline. In the following you can find an overview of these resources, for which modules you'd need them and what you'd need to do:
```

> 

Reviewer: ```suggestion
```

### Microsoft.Web/sites

> To successfully deploy the function app using the parameter file `fa.parameters.json` you need to create an Azure Active Directory App with API endpoint enabled (e.g. `api://<app id>`) and a secret added. The secret value needs then to be stored in a Key Vault secret.

Reviewer: ```suggestion
To successfully deploy the sites module using the parameter file `fa.parameters.json` you need to create an Azure Active Directory App with its API endpoint enabled (e.g. `api://<app id>`) and add a secret. The secret value needs then to be stored in a Key Vault secret.
```
